setroubleshoot 3.3.1                                       2015-08-18
 - switch and port setroubleshoot to Python 3

setroubleshoot 3.2.24                                      2015-04-23
 - updated translations from
   https://fedora.zanata.org/project/view/setroubleshoot

setroubleshoot 3.2.23                                      2015-04-09
 - setroubleshootd is set to be run as setroubleshoot user instead of
   root user
 - commands.getstatusoutput() is not used anymore
 - AppData file is provided now
 - bug fixes
